language/proto/kinds.go

```diff
@@ -19,9 +19,13 @@ import "github.com/bazelbuild/bazel-gazelle/rule"
 
 var protoKinds = map[string]rule.KindInfo{
 	"proto_library": {
-		NonEmptyAttrs:  map[string]bool{"srcs": true},
-		MergeableAttrs: map[string]bool{"srcs": true},
-		ResolveAttrs:   map[string]bool{"deps": true},
+		NonEmptyAttrs: map[string]bool{"srcs": true},
+		MergeableAttrs: map[string]bool{
+			"srcs":                true,
```

Comment:
Quoting the previous thread (I really wish GitHub would stop losing comments):

> The downside I see to making these attributes mergeable is repos that are already using the attributes will either have to start using the directives, or add a lot of # keep comments. The benefit of the attributes being mergeable is it makes it easier to mass edit them.

It's not clear to me which side of this we should be on. If these attributes already had widespread use, I'd say let's avoid modifying them if they're already set. But since they are new, maybe mergeable is okay. Still makes me nervous.

For reference, existing attributes can be bulk-deleted or modified with [buildozer](https://github.com/bazelbuild/buildtools/tree/master/buildozer). I don't think there's any way to bulk-add `# keep` comments though.